backend/orm_functions/session_key.ts

```diff
@@ -29,6 +29,7 @@ export async function addSessionKey(
  * @returns the loginUser associated with the key in a promise if the keys is valid otherwise an error in a promise
  */
 export async function checkSessionKey(key: string) {
+    console.log("key: " + key);
```

Comment:
The `console.log()` police is here! 🚨
Put down the log statement and come out with your hands above your head!